Formula/aom.rb

```diff
@@ -0,0 +1,58 @@
+class Aom < Formula
+  desc "Codec library for encoding and decoding AV1 video streams"
+  homepage "https://aomedia.googlesource.com/aom"
+  url "https://aomedia.googlesource.com/aom", :using => :git, :tag => "v1.0.0", :revision => "cb108fb20fec79291588f4925499ac4cb3e64bfd"
+
+  depends_on "cmake" => :build
+  depends_on "perl" => :build
+  depends_on "yasm" => :build
+
+  depends_on "ffmpeg" => :test
+
+  def install
+    args = %W[
+      -DCMAKE_INSTALL_PREFIX=#{prefix}
+      -DENABLE_EXAMPLES=on
+      -DENABLE_DOCS=off
+      -DENABLE_TESTDATA=off
+      -DENABLE_TESTS=off
+      -DENABLE_TOOLS=off
+    ]
+
+    mkdir "macbuild" do
```

Comment:
`mkdir "build"`

Reply:
That is not the best idea, the git repository (the source) already has a directory called "build" including some files. We would pollute the source that way.